### ISSUE TITLE
Won't launch in VS 2015

### ISSUE THREAD

**Issue by manu-st** [CodingWithCalvin/VS-OpenInNotepadPlusPlus#2]
I installed a few days ago the extension in VS 2019 preview and it worked great.
Today, I installed it in VS 2015, but there it doesn't work because of the following error:
```
 <entry>
    <record>599</record>
    <time>2019/03/11 20:42:42.097</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>CreateInstance failed for package [VsPackage]Source: &apos;mscorlib&apos; Description: Could not load file or assembly &apos;Microsoft.VisualStudio.Shell.15.0, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos; or one of its dependencies. The system cannot find the file specified.&#x000D;&#x000A;System.IO.FileNotFoundException: Could not load file or assembly &apos;Microsoft.VisualStudio.Shell.15.0, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos; or one of its dependencies. The system cannot find the file specified.&#x000D;&#x000A;File name: &apos;Microsoft.VisualStudio.Shell.15.0, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a&apos;&#x000D;&#x000A;   at System.Reflection.RuntimeAssembly.GetType(RuntimeAssembly assembly, String name, Boolean throwOnError, Boolean ignoreCase, ObjectHandleOnStack type)&#x000D;&#x000A;   at System.Reflection.RuntimeAssembly.GetType(String name, Boolean throwOnError, Boolean ignoreCase)&#x000D;&#x000A;   at System.Activator.CreateInstanceFromInternal(String assemblyFile, String typeName, Boolean ignoreCase, BindingFlags bindingAttr, Binder binder, Object[] args, CultureInfo culture, Object[] activationAttributes, Evidence securityInfo)&#x000D;&#x000A;   at System.AppDomain.CreateInstanceFrom(String assemblyFile, String typeName)&#x000D;&#x000A;&#x000D;&#x000A;WRN: Assembly binding logging is turned OFF.&#x000D;&#x000A;To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.&#x000D;&#x000A;Note: There is some performance penalty associated with assembly bind failure logging.&#x000D;&#x000A;To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].&#x000D;&#x000A;</description>
    <guid>{A8D3F4B6-CD76-47AB-A0DC-4E42869CEA9B}</guid>
    <hr>80004005 - E_FAIL</hr>
    <errorinfo></errorinfo>
  </entry>

```

Looks like the extension is incorrectly compiled. See https://stackoverflow.com/questions/46054343/visual-studio-2015-fail-to-load-package-because-microsoft-visualstudio-shell-15

Would it be possible to redo the extension so that it supports VS 215?

**Comment by CalvinAllen** [comment]
Oh my, did not realize that I had broken support for 2015.

Thanks for the link, that helps, I'm sure I upgraded those dependencies when I made the last set of changes, not realizing the effect it would have.

Let me make some changes and push a new version.  Hang tight, please! :)

**Comment by CalvinAllen** [comment]
@manu-st I just published 3.0.4 to the marketplace, let me know if you have any other issues.  Thanks!

**Comment by manu-st** [comment]
Thanks now the extension loads but it does not work. Creating a separate issue.
